qa/smoke-test-http/src/javaRestTest/java/org/elasticsearch/http/IndicesSegmentsRestCancellationIT.java

```diff
@@ -12,12 +12,23 @@
 import org.apache.http.client.methods.HttpGet;
 import org.elasticsearch.action.admin.indices.segments.IndicesSegmentsAction;
 import org.elasticsearch.client.Request;
+import org.elasticsearch.test.junit.annotations.TestIssueLogging;
 
 public class IndicesSegmentsRestCancellationIT extends BlockedSearcherRestCancellationTestCase {
+    @TestIssueLogging(
+        issueUrl = "https://github.com/elastic/elasticsearch/issues/88201",
+        value = "org.elasticsearch.http.BlockedSearcherRestCancellationTestCase:DEBUG"
+            + ",org.elasticsearch.transport.TransportService:TRACE"
+    )
```

Comment:
Is this test for comparison? The test failure ticket doesn't appear to mention `testIndicesSegmentsRestCancellation`

Reply:
Yes, it's unclear whether they're really different or not, and given that it's only failing once every few months it seems wise to gather more data here.